src/bin/server.ts

```diff
@@ -46,19 +55,44 @@ interface Args {
   if (configPath) {
     try {
       options = require(configPath);
-      // make sure that networkKey is passed as buffer.
-      // accept both zwave2mqtt format as ozw format
-      if (options.networkKey && options.networkKey.length === 32) {
-        options.networkKey = Buffer.from(options.networkKey, "hex");
-      } else if (options.networkKey && options.networkKey.includes("0x")) {
-        options.networkKey = options.networkKey
-          .replace(/0x/g, "")
-          .replace(/, /g, "");
-        options.networkKey = Buffer.from(options.networkKey, "hex");
-      } else {
-        console.error("Error: Invalid networkKey defined");
-        return;
+      // If both securityKeys.S0_Legacy and networkKey are defined, throw an error.
+      if (options.securityKeys?.S0_Legacy && options.networkKey) {
+        throw new Error(
+          "Both `networkKey` and `securityKeys.S0_Legacy` options are both present in the " +
+            "config and we only need one. `securityKeys.S0_Legacy` is preferred over `networkKey` " +
+            "and should match your old `networkKey` if you were using it before."
+        );
       }
+      const securityKeyNames = [
+        "S0_Legacy",
+        "S2_AccessControl",
+        "S2_Authenticated",
+        "S2_Unauthenticated",
+      ];
+      // We prefer the securityKeys option over the networkKey one
+      if (options.securityKeys)
+        for (const key of securityKeyNames)
+          if (key in options.securityKeys)
+            options.securityKeys[key] = normalizeKey(
+              options.securityKeys[key],
+              `securityKeys.${key}`
+            );
```

Comment:
```suggestion
      if (options.securityKeys) {
        for (const key of securityKeyNames) {
          if (key in options.securityKeys) {
            options.securityKeys[key] = normalizeKey(
              options.securityKeys[key],
              `securityKeys.${key}`
            );
          }
        }
      }
```

Reply:
btw, this means that we allow normalizing all keys. Is the fallback something we want to offer for new options ?

Reply:
There's a third parameter that controls whether OZW format is supported. For now, I have not added support for that for the new options (defaults to false)